go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
 	github.com/gardener/etcd-druid v0.5.0
-	github.com/gardener/gardener v1.24.1-0.20210608063816-580010846480
+	github.com/gardener/gardener v1.24.1-0.20210611132013-a6aa5c0904a9
```

Comment:
I assume, you will cherry-pick this and vendor `v1.23.2` in the cherry-pick PR, right?

Reply:
Yes, https://concourse.ci.gardener.cloud/teams/gardener/pipelines/gardener-release-v1.23/jobs/release-v1.23-release-job/builds/4 is running and I'll open a PR for `release-v1.25` once `gardener/gardener@v1.23.2` is released successfully :)

Reply:
Great 🚀